strutil/strutil.go

```diff
@@ -40,3 +42,38 @@ func JSON(v interface{}) string {
 	bytes, _ := json.Marshal(v)
 	return string(bytes)
 }
+
+func wildCardToRegexp(pattern string) string {
+	var result strings.Builder
+	for i, literal := range strings.Split(pattern, "*") {
+
+		// Replace * with .*
+		if i > 0 {
+			result.WriteString(".*")
+		}
+
+		// Quote any regular expression meta characters in the
+		// literal text.
+		result.WriteString(regexp.QuoteMeta(literal))
+	}
+	return result.String()
+}
+
+func regexMatch(pattern string, value string) bool {
+	result, _ := regexp.MatchString(wildCardToRegexp(pattern), value)
+	return result
+}
+
+func CheckAllExclude(excludePatterns []string, urlPath string) bool {
```

Comment:
We might find a better name for this function, IMO. Like `IsURLExcluded` or `IsFileExcluded`

Reply:
changed to `IsURLExcluded`.